docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -1,116 +1,101 @@
-#  How Do I Check my Resource Usage on the Yens?
-If you would like to check your resource usage on the Yens, there are a few tools available.
+# How Do I Check My Resource Usage on the Yens?
+Monitoring your resource usage is important to be a good citizen of the GSB Research Computing community, but also to be able to effectively estimate your needs on scheduled systems.
 
-- `htop` allows you to get an overview of the activity on the whole system that you are logged into. Furthermore, you can inspect your own processes by typing `htop -u <SUNetID>`. A lot more information about how to decipher the information produced with this command can be found on [this page](/yen/community.html).
-- `topbyuser` is another tool written by us that lists out the active users on the Yen you are logged into and the amount of resources they are currently using.
+These tools can help you characterize your usage:
 
-!!! important
-    We have [community guidelines](/yen/community.html) to illustrate responsible use of our shared resources that our users are expected to follow. These guidelines ensure that the servers can remain accessible to everyone.
+- **`htop`** allows you to get an overview of the activity on the whole system that you are logged into. 
+- **`userload`** is a custom tool that allows you to monitor CPU and RAM usage in a simplified view.
+- **`topbyuser`** is another custom tool that lists out the active users on your current host and the amount of resources they are currently using.
 
+!!! important
+    We have [Community Guidelines](/_policies/user_limits){target="_blank"} to illustrate responsible use of our shared resources that our users are expected to follow. These guidelines ensure that the Yens can remain accessible and performant.
 
-## Other page
-##  Monitoring Usage
-## Monitoring Your Resource Footprint
+## Monitoring Compute with `htop`
+One easy method of getting a quick snapshot of your CPU and memory usage is via the `htop` command line tool. Running `htop` shows usage graphs and a process list that is sortable by user, top CPU, top RAM, and other metrics. Please use this tool liberally to monitor your resource usage, especially if you are running multiprocessing code on shared systems for the first time.
 
-Certain parts of the GSB research computing infrastructure provide
-environments that are managed by a scheduler (like <a href="_policies/services" target="_blank">Sherlock</a> or <a href="/yen/scheduler.html" target="_blank">Yen-Slurm</a>). In these cases it is not necessary for individuals to monitor resource usage themselves.
+You can toggle between a threaded view of processes by pressing ++t++ -- here is sample `htop` output in threaded mode.
```

Comment:
feel a bit lengthy for a sentence? Maybe we can split up to say "You can toggle to a threaded view of processes by pressing ++t++. Below is a sample of `htop` output in threaded mode."